app/views/entities/show.html.erb

```diff
@@ -0,0 +1,21 @@
+
+  <div class="col-lg-10 col-md-10 col-sm-10 mx-auto">
+      <div class="card mb-2 shadow">
+        <div class="card-body">
+          <div class="d-flex gap-2 align-items-center ml-auto category-card">
+              <div class="category_name">
+                <h2> <%= @entity.name %></h2>
+              </div>
+              <div class="category_amount">
+                <p><%= number_to_currency( @entity.amount) %></p>
+              </div>
+          </div>
+        </div>
+      </div>
+      <div>
+        <%= link_to "Edit this entity", edit_user_entity_path(@user, @entity) %> |
+        <%= link_to "Back to entities", user_entities_path %>
+
+        <%= button_to "Destroy this entity", user_entity_path, method: :delete %>
+      </div>
+  </div>
```

Comment:
- Hey, great job adding this functionality, however, when user tries to destroy it, it throws an error. Since this is not a requirement, you can remove it or fix it to improve the user experience as right now, it makes your app break. Can you work on this, please?



![image](https://github.com/zbahati/budgetApp/assets/59397159/e7461b1f-7004-4201-84c8-028725331164)